lib/routemap.h

```diff
@@ -135,8 +138,6 @@ enum rmap_compile_rets {
 	/* Route map rule can't compile */
 	RMAP_COMPILE_ERROR,
 
-	/* Route map rule is duplicate */
-	RMAP_DUPLICATE_RULE
```

Comment:
nit: could you explain just why we remove this return type? won't be used anymore?

Reply:
This was introduced in the context of below PR : 
https://github.com/FRRouting/frr/pull/4345

Now we have moved the dependency table update(route_map_upd8_dependency) inside the routemap lib.
So it's not required.